pelicanconf.py

```diff
@@ -1,32 +1,39 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*- #
-from __future__ import unicode_literals
 
-AUTHOR = u'CON team'
-SITENAME = u'CON'
+AUTHOR = 'CON team'
+SITENAME = 'CON'
 SITEURL = ''
 
-TIMEZONE = 'Americas/Eastern'
+TIMEZONE = 'America/New_York'
 
-DEFAULT_LANG = u'en'
+DEFAULT_LANG = 'en'
 
 DIRECT_TEMPLATES = ['index']
 
 STATIC_PATHS = [ 'img', 'CNAME' ]
 
 THEME = 'theme'
 
-MD_EXTENSIONS = [ 'codehilite(css_class=highlight)', 'extra', 'mdext.fix_code_blocks' ]
+MARKDOWN = {
+    'extension_configs': {
+        'markdown.extensions.codehilite': {'css_class': 'highlight'},
+        # TODO(asmacdo) I assumed we had to declare fix_code_blocks here, but it doesn't make a change.
```

Comment:
@yarikoptic :eyes:

Reply:
Note: that there might be lots of features we don't use. This website/setup was borrowed from the  https://github.com/buddycloud/buddycloud.com (http://buddycloud.com/) and many pages/features disabled, so some settings might not even be pertinent to CON website.